src/routes/quotes/data.json

```diff
@@ -1,25 +1,139 @@
+    "author": "Donald Trump",
+    "author_description": "45th President of the United States",
+    "color": "white",
+    "padding": "0rem"
+  },
+  {
+    "background": "biden",
+    "text": "Social media has shown us the harm that powerful technology can do without the right safeguards in place. [...] we must be clear-eyed and vigilant about the threats emerging — of emerging technologies that can pose — don’t have to, but can pose — to our democracy and our values. [...] Realizing the promise of AI by managing the risk is going to require some new laws, regulations, and oversight.",
+    "author": "Joe Biden",
+    "author_description": "46th President of the United States",
+    "color": "white"
+  },
+  {
+    "background": "putin",
+    "text": "Artificial intelligence is the future, not only for Russia, but for all humankind. It comes with colossal opportunities, but also threats that are difficult to predict. Whoever becomes the leader in this sphere will become the ruler of the world.[...] If we become leaders in this area, we will share this know-how with entire world, the same way we share our nuclear technologies today",
+    "author": "Vladimir Putin",
+    "author_description": "President of Russia",
+    "color": "white"
+  },
+  {
+    "background": "qiang",
+    "text":"AI must be guided in a direction that is conducive to the progress of humanity. So there should be a red line in AI development, a red line that must not be crossed. [...] It should not just benefit only a small group of people, but benefit the overwhelming majority of mankind. [...] It is essential that we work together and coordinate with each other.",
+    "author": "Li Qiang",
+    "author_description": "China's Head of Government",
+    "color": "black"
+  },
+  {
+    "background": "leyen",
+    "text": "[We] should not underestimate the real threats coming from AI. [Fully quoted the above statement on the risk of extinction.] [...] It is moving faster than even its developers anticipated. [...] We have a narrowing window of opportunity to guide this technology responsibly.",
+    "author": "Ursula von der Leyen",
+    "author_description": "Head of the executive branch of the European Union",
+    "color": "white"
+  },
+  {
+    "background": "guterres",
+    "text": "AI poses a long-term global risk. Even its own designers have no idea where their breakthrough may lead. I urge [the UN Security Council] to approach this technology with a sense of urgency. Unforeseen consequences of some AI-enabled systems could create security risks by accident. Generative AI has enormous potential for good and evil at scale. Its creators themselves have warned that much bigger, potentially catastrophic and existential risks lie ahead. Without action to address these risks, we are derelict in our responsibilities to present and future generations.",
```

Comment:
This one might be too long too?  Would be more readable if the text didn't overlap with the image?

<img width="337" alt="image" src="https://github.com/joepio/pauseai/assets/61096713/6bbb60db-858b-430e-adb1-9fd0c1f59f25">